### DISCUSSION TITLE
Example of how to use host docker image in case of DIND run (as it was in 1.5.0)

### DISCUSSION THREAD

**Discussion by ipetrushevskiy** [testcontainers/testcontainers-dotnet#434] [Q&A] [answered]
Hello.
Were using 1.5.0 version where TestContainers runned inside the container were using host image instead of downloading it into the volume.
Is it possible in current beta version the same behavior?

Thank you in advance.

**Comment by HofmeisterAn** [comment, marked as answer]
I'm sorry, I don't understand what you're talking about. Can you add more information? Are you talking about a feature that works with `1.5.0`, but not with the latest snapshot release?

**Reply by ipetrushevskiy** [reply]
Hey, np - I was quite vague, agree.

The thing is - I am using C# app with the test-containers as is and sometimes - inside another container (so the test-containers utilize docker in docker approach in that case).
The difference between 1.5.0 and 1.6.x is that in 1.5.0 - both scenarios were pulling and running test-containers image on the host docker machine (so if I were recreating container with the C# app i didn't have to pull the test-containers image again)
in 1.6.0 - it is pulling the image "inside" - into the volume I guess? host image is not required anymore an not running.

So the question is - could I Fallback to the previous behavior?

**Reply by HofmeisterAn** [reply]
Ok, got you. Right now, I'm not aware of any change. I think it should still behave the same. I can probably set up a test this weekend.

**Reply by ipetrushevskiy** [reply]
Hey. Double checked - sorry for confusing everyone. Were a bit dizzy this days...
Test-containers behavior didn't changed indeed - but for some reason I assumed that what I described is how it should work...
To achieve desired by me behavior the mount could be specified like this:
`-v /var/run/docker.sock:/var/run/docker.sock`

**Reply by HofmeisterAn** [reply]
Yes, mounting the Docker socket is necessary in that case! Use `WithBindMount("/var/run/docker.sock", "/var/run/docker.sock", AccessMode.ReadOnly)`. This allows access to the host images. If that solves your issue, please, mark this as an answer.
